lib/squib/graphics/shapes.rb

```diff
@@ -18,8 +18,15 @@ def rect(box, draw, trans)
     def circle(box, draw)
       x, y, r = box.x, box.y, box.radius
       use_cairo do |cc|
-        cc.move_to(x + r, y)
-        cc.circle(x, y, r)
+        # cc.move_to(x + r, y)
```

Comment:
I needed to remove this to avoid drawing a line from the end of the arc to this point. If this cursor movement is necessary for some other purpose, I will need to figure out how to accomplish both needs.